pkg/util/policy/agent/agent.py

```diff
@@ -56,6 +47,9 @@ def filter_rules_idx(rules):
 # We want to receive json object as a POST.
 class AgentHandler(BaseHTTPRequestHandler):
     def do_GET(self):
+        """
+        We do not have a storage so GET is not really supported.
```

Comment:
No, it wasn't.

Iptables is a shared storage - anyone can place anything in it, I think it would be a mistake to rely on substring search in public storage. 

Unless we should really do this.